src/email/mail_cmd.rs

```diff
@@ -0,0 +1,49 @@
+/**
+ * Implementation for default "mail" command in linux
+ */
+use super::email_provider::EmailProvider;
+
+#[derive(Default)]
+pub struct MailCommand {}
+
+impl EmailProvider for MailCommand {
+  fn send_email(&self, address: &str, contents: &str) {
+    send_email(address, contents);
+  }
+}
+
+const TEMPORARY_FILE_NAME : &str = "/tmp/rss2-email.txt";
+
+
+#[cfg(not(target_os = "windows"))]
+fn send_email(address: &str, contents: &str) {
+  use log::{info,warn};
+  use std::{fs::File, io::Write, process::Command};
+
+  let mut file =
+    File::create(TEMPORARY_FILE_NAME).expect("Can't create temporary file /tmp/rss2-email.txt");
+  file
+    .write_all(contents.as_bytes())
+    .expect("Failed to write temporary email");
+
+  let mail_command = format!("mail -s \"Rss2Email\" \"{address}\" < /tmp/rss2-email.txt");
```

Comment:
The `TEMPORARY_FILE_NAME` should be used in the `expect` and `format` calls in lines 24 and 29 here.

Reply:
I'll remove it from line 24, because concatenation/formatting in error messages triggers clippy about "function call in expect()" .

The other one I missed!